lib/tasks/capitalize-recipe-name.rake

```diff
@@ -0,0 +1,25 @@
+desc "capitalize recipe name after special characters"
+task :capitalize_name => :environment do
+  Recipe.where("name LIKE ? or name LIKE ? or name LIKE ?", '%-%', '%(%', '%"%').find_each(batch_size: 100) do |recipe|
+
+    special_characters = ['-', '\(', '"']
+    name = capitalize_str(special_characters, recipe.name)
+    puts name
+    
+    Recipe.find(recipe.id).update(name: name)
+  end
+end
+
+def capitalize_str(special_characters, name)
+  special_characters.each do |character|
```

Comment:
I think the loop shouldn't be part of the function. A function should just do a single task and be re-usable.